core/app/models/spree/payment.rb

```diff
@@ -145,6 +145,11 @@ def can_credit?
       credit_allowed > 0
     end
 
+    # @return [Boolean] true when this payment has been fully refunded
+    def fully_refunded?
+      refunds.sum(:amount) >= amount
```

Comment:
should `==` instead of `>=` be sufficient? If you refunded more then the amount paid there are some other issues going on IMHO.